pkgs/by-name/fa/fastcap/package.nix

```
@@ -0,0 +1,94 @@
+{ stdenv
+, fetchzip
+, ghostscript
+, lib
+, texliveMedium
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fastcap";
+  version = "2.0-18Sep92";
+
+  src = fetchzip {
+    url = "http://www.rle.mit.edu/cpg/codes/fastcap-${version}.tgz";
```

```suggestion
    url = "https://www.rle.mit.edu/cpg/codes/fastcap-${version}.tgz";
```
the site offers https and we should use it

---

Agreed. I'll change that